Bixal/move.mil
# Feature/offices insterter

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

Adds functionality to the PPM_tools to read the offices .json files and inserts the database into the database as content of type 'Location'.

## Testing

- Pull the branch.
- Install the module.
- Run  'docker-compose run php drupal parser'
- Check the nodes being made.
## Screenshots

_Attach relevant before and after screenshots here._
